superset/key_value/commands/update.py

```diff
@@ -87,7 +87,6 @@ def update(self) -> Optional[Key]:
             entry.expires_on = self.expires_on
             entry.changed_on = datetime.now()
             entry.changed_by_fk = get_user_id()
-            db.session.merge(entry)
```

Comment:
Do you know if the `autoflush(False)` when querying for the model influences anything?

Reply:
@michael-s-molina I removed the unnecessary auto-flush logic in https://github.com/apache/superset/pull/26009.